components/central-application-gateway/internal/proxy/reverseproxy.go

```diff
@@ -55,12 +68,31 @@ func newProxy(targetURL string, requestParameters *authorization.RequestParamete
 			setCustomHeaders(req.Header, requestParameters.Headers)
 		}
 
-		log.Infof("Modified request url : '%s', schema : '%s', path : '%s'", req.URL.String(), req.URL.Scheme, req.URL.Path)
+		zap.L().Info("modified request URL",
+			zap.String("url", req.URL.String()),
+			zap.String("schema", req.URL.Scheme),
+			zap.String("path", req.URL.Path))
 	}
+
+	log := zap.L().Sugar()
```

Comment:
As far as I understand, Suger() version has a worse performance. It's hard for me to determine which parts of the code are performance critical - have you considered performance criticality in the places you've "sugared" (😄) the logs?